<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>424</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>559</width>
    <height>424</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>559</width>
    <height>424</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="SegmentedWidget" name="MenuWidget">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>33</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PopUpAniStackedWidget" name="MainWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="Onepage">
      <widget class="SubtitleLabel" name="SubtitleLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>361</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>欢迎使用 HLDDZ_FullAuto 配置向导</string>
       </property>
      </widget>
      <widget class="PrimaryPushButton" name="StartButton">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>330</y>
         <width>153</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
      <widget class="ImageLabel" name="OneImage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>72</width>
         <height>72</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Twopage">
      <widget class="PrimaryPushButton" name="InstallPythonButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>153</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>安装 Python</string>
       </property>
      </widget>
      <widget class="PrimaryPushButton" name="Install3PacketButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>191</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>安装 Python 第三方库</string>
       </property>
      </widget>
      <widget class="SubtitleLabel" name="SubtitleLabel_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>361</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>第一步 配置环境</string>
       </property>
      </widget>
      <widget class="CaptionLabel" name="CaptionLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>1. 安装 Python</string>
       </property>
      </widget>
      <widget class="CaptionLabel" name="CaptionLabel_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>441</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>点击按钮时，会自动下载Python 安装程序，等待下载完成后请手动安装Python</string>
       </property>
      </widget>
      <widget class="CaptionLabel" name="CaptionLabel_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="CaptionLabel" name="CaptionLabel_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>391</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>点击按钮时，会自动下载Python 第三方库，请取消代理，耐心等待</string>
       </property>
      </widget>
      <widget class="CaptionLabel" name="CaptionLabel_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>2. 安装 Python 第三方库</string>
       </property>
      </widget>
      <widget class="CaptionLabel" name="CaptionLabel_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>301</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>安装 Python 时，请务必勾选 Add Python3.8 to PATH</string>
       </property>
      </widget>
      <widget class="PrimaryPushButton" name="OneNextButton">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>330</y>
         <width>153</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>下一步</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Threepage">
      <widget class="SubtitleLabel" name="SubtitleLabel_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>361</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>第二步 设置识图位置</string>
       </property>
      </widget>
      <widget class="ComboBox" name="SelectPosComboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>291</width>
         <height>32</height>
        </rect>
       </property>
      </widget>
      <widget class="PrimaryPushButton" name="SelectPosButton">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>220</y>
         <width>111</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>选区</string>
       </property>
      </widget>
      <widget class="PushButton" name="PrintScreenButton">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>220</y>
         <width>102</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>截图</string>
       </property>
      </widget>
      <widget class="CaptionLabel" name="CaptionLabel_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>3. 选区设置</string>
       </property>
      </widget>
      <widget class="CaptionLabel" name="CaptionLabel_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>421</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>您可以选择将显示器分辨率设置为 1920*1200 ，游戏全屏来使用默认选区参数</string>
       </property>
      </widget>
      <widget class="PrimaryPushButton" name="ShowDisplaySetButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>151</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>跳转至显示设置</string>
       </property>
      </widget>
      <widget class="CaptionLabel" name="CaptionLabel_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>421</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>您也可以手动选取识图选区，依次选取项目进行选区操作</string>
       </property>
      </widget>
      <widget class="PushButton" name="ClearButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>121</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>清空所有项目</string>
       </property>
      </widget>
      <widget class="PushButton" name="OpenSelectPosButton">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>180</y>
         <width>121</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>打开选区数据</string>
       </property>
      </widget>
      <widget class="PushButton" name="CopySelectPosButton">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>180</y>
         <width>131</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>复制选区数据</string>
       </property>
      </widget>
      <widget class="PushButton" name="OpenMainButton">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>180</y>
         <width>121</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>打开 main.py</string>
       </property>
      </widget>
      <widget class="CaptionLabel" name="CaptionLabel_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>421</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>选取所有选区后，请复制选区数据，将 main.py 中的参数部分进行替换</string>
       </property>
      </widget>
      <widget class="PrimaryPushButton" name="TwoNextButton">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>330</y>
         <width>153</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>下一步</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Fourpage">
      <widget class="PushButton" name="DoneButton">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>330</y>
         <width>121</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>完成</string>
       </property>
      </widget>
      <widget class="PrimaryPushButton" name="OpenProgramButton">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>330</y>
         <width>153</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>打开主程序</string>
       </property>
      </widget>
      <widget class="SubtitleLabel" name="SubtitleLabel_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>461</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>恭喜，您已完成HLDDZ_FullAuto 大部分配置！</string>
       </property>
      </widget>
      <widget class="ImageLabel" name="FourImage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>72</width>
         <height>72</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
      <widget class="CaptionLabel" name="CaptionLabel_31">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>打开HLDDZ_FullAuto尽情享受吧</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PopUpAniStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CaptionLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SubtitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>Pivot</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SegmentedWidget</class>
   <extends>Pivot</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
